src/xrpld/rpc/handlers/ServerInfo.cpp

```diff
@@ -235,6 +235,19 @@ ServerDefinitions::ServerDefinitions() : defs_{Json::objectValue}
         defs_[jss::FIELDS][i++] = a;
     }
 
+    {
+        Json::Value a = Json::arrayValue;
+        a[0U] = "index";
+        Json::Value v = Json::objectValue;
+        v[jss::nth] = 258;
+        v[jss::isVLEncoded] = false;
+        v[jss::isSerialized] = false;
+        v[jss::isSigningField] = false;
+        v[jss::type] = "Hash256";
+        a[1U] = v;
+        defs_[jss::FIELDS][i++] = a;
+    }
```

Comment:
I think it's better to just re-add the SField macro definition, if that would work.

Reply:
There are other hard-coded fields, so this isn't alone. See right above this.

Reply:
Happy to change it if you want me to, though.

Reply:
I'd rather change it back. I know there are other special fields, but I would love to see `sfInvalid` and `sfGeneric` treated like other SFields, honestly. I removed `sfIndex` because I thought it was unused. Apparently, it is used, but there is no testament to that in the code or tests. Thus, it was mistakenly removed. I'd rather restore the definition and add a test for it.